### PR TITLE
Up81

### PR DESCRIPTION
**Pull request by Ryan-Guinn**

### CODE REVIEW

File: packages/ui-bindings/Plan/planInterface.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  name: string
+  plannedOn: string
+  due: string
+  note: string
+  scope: [Agent]
+  planProcesses: [Process]
+  workingAgents: [Agent]
+  plannedNonWorkInputs: any // TODO: replace with [Commitment]
+  plannedOutputs: any // TODO: replace with [Commitment]
+  nonWorkInputs: any // TODO: replace with [EconomicEvent]
```
Comment (fontainesw) [inline packages/ui-bindings/Plan/planInterface.tsx]: Economic event already exists in the project, but It dosen't have a type defined. you should define it in the interface file for economic event, and then use that here

File: packages/app/routes.tsx [hunk 2]
```diff
@@ -65,6 +66,7 @@ const routes = (
       <Route path="agentrelationshiprole" component={AgentRelationshipRoles} />
       <Route path="agentrelationship" component={AgentRelationship}/>
       <Route path="process" component={Process}/>
+      <Route path="plan" component={Plan}/>
       <Route path="person" component={PersonPage} />
     </Route>
   </Route>
```
Comment (fontainesw) [inline packages/app/routes.tsx:72]: The "export default routes" needs a semicolon

**Review by fontainesw — COMMENTED** [review]
Looks pretty good, I just have 2 things

File: packages/app/routes.tsx (hunk 2 above)
Comment (Ryan-Guinn) [reply packages/app/routes.tsx:72]: fixed

File: packages/ui-bindings/Plan/planInterface.tsx (hunk 1 above)
Comment (Ryan-Guinn) [reply packages/ui-bindings/Plan/planInterface.tsx]: added

**Review by fontainesw — APPROVED** [review]

File: packages/ui-bindings/EconomicEvent/economicEventInterface.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (ConnorHibbs) [inline packages/ui-bindings/EconomicEvent/economicEventInterface.tsx:1]: Is this a new file? There is already a file with this same name and path in Master that defined the GraphQL fragment for EconomicEvent.

**Review by ConnorHibbs — APPROVED** [review]
Looks pretty good. I had a question about EconomicEvent.tsx but code wise everything is good.

**Review by Lou3797 — APPROVED** [review]

**Pull request merged**